src/handlers/payout/action.ts

```diff
@@ -88,10 +89,26 @@ export const handleIssueClosed = async () => {
   }
 
   logger.info(`Handling issues.closed event, issue: ${issue.number}`);
-  if (paymentPermitMaxPrice == 0) {
+  for (const botComment of comments.filter((cmt) => cmt.user.type === UserType.Bot).reverse()) {
+    const botCommentBody = botComment.body;
+    if (botCommentBody.includes(GLOBAL_STRINGS.autopaycomment)) {
+      const pattern = /\*\*(\w+)\*\*/;
+      const res = botCommentBody.match(pattern);
+      if (res) {
+        if (res[1] === "false") {
+          logger.info(`Skipping to generate permit2 url, reason: autoPayMode for this issue: false`);
+          return `Permit generation skipped since autoPayMode for **THIS ISSUE** is disabled`;
```

Comment:
```suggestion
          return `Permit generation skipped since autoPayMode for ${issue} is disabled`;
```
I'm not sure what the correct variable is but shouldn't it be specific for the log?

Reply:
It is the comment for informing that autopay is disabled for a specific issue.
It's not a log.
So, it does more make sense to do that.